### DISCUSSION TITLE
MSFS touch portal plugin

### DISCUSSION THREAD

**Discussion by Tankusek** [mpaperno/MSFSTouchPortalPlugin#65] [Q&A] [answered]
Hi today i downloaded you plugin but unfortunately is not working and i dont know why.
![image](https://github.com/mpaperno/MSFSTouchPortalPlugin/assets/47474545/798ce6e7-2f65-48d3-8b36-aa57ac66e30f)
![image](https://github.com/mpaperno/MSFSTouchPortalPlugin/assets/47474545/15fbc01a-f173-4394-b413-13ed2b4b64f3)
Thats my button to establish connection.
...
and heres example of my button
![image](https://github.com/mpaperno/MSFSTouchPortalPlugin/assets/47474545/316a3b70-3805-4767-941b-40cc4d2468d7)
![image](https://github.com/mpaperno/MSFSTouchPortalPlugin/assets/47474545/8af5194d-4245-4ab1-97d2-5ff557e929fa)
I was trying to reconnect, restart PC and plugin and nothing have changed.
Any idea what am i doing wrong ?

**Comment by mpaperno** [comment]
Hi,

Could you provide some more information about what you've tried and what exactly isn't working?  I'd like to help but I can't determine anything from just two screenshots of standard buttons from the example page.

Can you connect to the sim?  Is the plugin running?  Is it installed?

Check your antivirus software if the plugin isn't even running.

Check that the plugin's log file is there and see what it says.  If it's not there, the plugin isn't starting at all.

See:  https://github.com/mpaperno/MSFSTouchPortalPlugin#troubleshooting

Thanks,
-Max

**Reply by Tankusek** [reply]
> Can you connect to the sim? Is the plugin running? Is it installed?<

According to button i copied from page icnluded on github, plugin should be connected
![image](https://github.com/mpaperno/MSFSTouchPortalPlugin/assets/47474545/abca85a6-080f-41c9-bd42-80d6f190ce0b)
![image](https://github.com/mpaperno/MSFSTouchPortalPlugin/assets/47474545/f62ab903-8e6e-469f-8ade-cb55b3de8742)
It changing color to green so i assume it is connected, however... <

> Check that the plugin's log file is there and see what it says. If it's not there, the plugin isn't starting at all.<

In app settting it seems that plugin is installed an running 
![image](https://github.com/mpaperno/MSFSTouchPortalPlugin/assets/47474545/4281b740-66b6-41d2-a7f9-cd2e281cfced)

> Check that the plugin's log file is there and see what it says. If it's not there, the plugin isn't starting at all.<

According to theese three files sometimes its connected sometimes it isnt


[MSFSTouchPortalPlugin20231201.log](https://github.com/mpaperno/MSFSTouchPortalPlugin/files/13600321/MSFSTouchPortalPlugin20231201.log)

[MSFSTouchPortalPlugin20231202.log](https://github.com/mpaperno/MSFSTouchPortalPlugin/files/13600326/MSFSTouchPortalPlugin20231202.log)

[MSFSTouchPortalPlugin20231203.log](https://github.com/mpaperno/MSFSTouchPortalPlugin/files/13600334/MSFSTouchPortalPlugin20231203.log)

[MSFSTouchPortalPlugin20231207.log](https://github.com/mpaperno/MSFSTouchPortalPlugin/files/13600341/MSFSTouchPortalPlugin20231207.log)

[WASimClient.log](https://github.com/mpaperno/MSFSTouchPortalPlugin/files/13600354/WASimClient.log)

> Check your antivirus software if the plugin isn't even running.
i'm using default system anti virus and of course i launch app as administrator <

Hopefully you can help me with that im realy looking forward to it.
It is great plugin which can help me a lot

thank you

**Comment by mpaperno** [comment, marked as answer]
Solved via Discord chat, IIRC. Closing here.
